### DISCUSSION TITLE
Hit crates.io rate limit during v0.50.0 release

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#3173] [General]
Just a shout into the void in case this is useful in any way. During the v0.50.0 release I hit the crates.io rate limit of max 30 crates per minute (https://github.com/rust-lang/crates.io/issues/1643).

My workaround was to do two rounds of publishing, one with 30 crates and one with the remaining.

I think this is fine. I don't think we need to reduce the number of crates that we have due to this.

**Comment by thomaseizinger** [comment]
> I think this is fine. I don't think we need to reduce the number of crates that we have due to this.

Ideally, we reduce the number of version bumps that trickle up the dependency chain so we can release more crates individually without triggering a release for the entire workspace.
